com.onepassword.OnePassword.yml

```diff
@@ -0,0 +1,54 @@
+app-id: org.onepassword.onepassword
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: 1password
+rename-desktop-file: 1password.desktop
+rename-icon: 1password
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-config
+  - --device=all
+modules:
+  - name: 1password
+    buildsystem: simple
+    build-commands:
+      - tar -xf 1password-latest.tar.gz
+      - mkdir -p /app/share/icons
+      - mkdir -p /app/1Password
+      - install -Dt /app/share/applications 1password-8.10.8.x64/resources/1password.desktop
+      - sed -i 's/Exec=.*/Exec=1password %U/' /app/share/applications/1password.desktop
```

Comment:
please use `destkop-file-edit` for that.

Reply:
Can you give me more information about this, I can't find how to correct it.

Reply:
`man desktop-file-edit`

It will explain better than I can.